routes/web.php

```diff
@@ -230,6 +230,14 @@
 
             // project messages
             Route::resource('{project}/messages', 'Company\\Project\\ProjectMessagesController', ['as' => 'projects']);
+
+            // project tasks
+            Route::resource('{project}/tasks', 'Company\\Project\\ProjectTasksController', ['as' => 'projects']);
+            Route::post('{project}/tasks/{task}/toggle', 'Company\\Project\\ProjectTasksController@toggle');
+            Route::get('{project}/tasks/lists/create', 'Company\\Project\\ProjectTaskListsController@create');
+            Route::post('{project}/tasks/lists/store', 'Company\\Project\\ProjectTaskListsController@store');
+            Route::put('{project}/tasks/lists/{list}', 'Company\\Project\\ProjectTaskListsController@update');
+            Route::delete('{project}/tasks/lists/{list}', 'Company\\Project\\ProjectTaskListsController@destroy');
```

Comment:
this should also be able to be written:
```suggestion
            Route::resource('{project}/tasks/lists', 'Company\\Project\\ProjectTaskListsController')-only([
               'create', 'store', 'update', 'destroy'
            ]);
```

Reply:
Done!